### PR TITLE
Remove unnecessary @webcomponents/custom-elements package

### PR DESCRIPTION
**Pull request by MRafaydev**
** Describe the change **
This pull request removes the library "@webcomponents/custom-elements", which was added as a polyfill for the `window.customElements` API for older versions of Firefox ESR. According to https://developer.mozilla.org/en-US/docs/Web/API/Window/customElements, the current version of Firefox ESR (102) now supports this API natively, so the library is no longer needed and can be removed.

** Issue reference **
Fixes https://github.com/kiali/kiali/issues/5805

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (jmazzitelli) [inline yarn.lock:1]: This file should not be here.

File: node_modules/.yarn-integrity [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (jmazzitelli) [inline node_modules/.yarn-integrity:1]: What is this file? I do not think this should be here.

**Comment by jmazzitelli** [comment]
@MRafaydev The frontend code of Kiali is in the [frontend/ directory](https://github.com/kiali/kiali/tree/master/frontend)  -- the fact that `node_modules` and `yarn.lock` are in your root Kiali directory that you pushed in this PR means your dev set up is not correct (those files are in `frontend/` and in Kiali project we do not check in `node_modules` at all - those should be git ignored as you see [here](https://github.com/kiali/kiali/blob/master/frontend/.gitignore#L4)).

Before submitting PRs against the Kiali code base, it is best to become familiar with it and its development environment. I suggest reading through and understanding these files:

- https://github.com/kiali/kiali/blob/master/README.adoc#developer-setup
- https://github.com/kiali/kiali/blob/master/frontend/README.adoc

If you see an issue that you would like to try to fix, feel free to submit a [Q&A discussion on the topic](https://github.com/kiali/kiali/discussions) and the Kiali team can help walk you through it.

File: yarn.lock (hunk 1 above)
Comment (jshaughn) [reply yarn.lock:1]: These files show up if you accidentally execute `yarn install` at the top level `kiali` directory.

**Comment by MRafaydev** [comment]
As a beginner, I apologize for making this mistake and will make sure to
follow the proper development setup as outlined in the provided links. I
appreciate your guidance and will make sure to review the Kiali development
environment before submitting any future PRs. Thank you for taking the time
to point out this error and help me improve my contributions to the project.

File: Makefile [hunk 3]
```diff
@@ -29,7 +29,7 @@ VERSION_LABEL ?= ${VERSION}
 # The go commands and the minimum Go version that must be used to build the app.
 GO ?= go
 GOFMT ?= $(shell ${GO} env GOROOT)/bin/gofmt
-GO_VERSION_KIALI = 1.18.7
+GO_VERSION_KIALI = 1.19.5
```
Comment (ferhoyos) [inline Makefile:32]: This change does not correspond to the purpose of the PR, but to #5800 . Please try to keep separate branches when working on multiple PRs at the same time.

**Review by ferhoyos — CHANGES_REQUESTED** [review]
Also you need to remove @webcomponents/custom-elements reference in polyfills.ts file

File: frontend/src/polyfills.ts [hunk 4]
```diff
@@ -1,3 +1,2 @@
 // Polyfills window.customElements; currently required for Firefox ESR
```
Comment (ferhoyos) [inline frontend/src/polyfills.ts:1]: Please remove the comments too.

File: go.mod [hunk 5]
```diff
@@ -1,6 +1,6 @@
 module github.com/kiali/kiali
 
-go 1.18
+go 1.19
```
Comment (jmazzitelli) [inline go.mod:3]: this change does not belong in this PR. It is in the other PR branch.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
`frontend/yarn.lock` is deleted but should not be. This needs to remain.

This PR really needs to be cleaned up before review/acceptance.

**Comment by MRafaydev** [comment]
Can i raise the new clean PR ?

**Comment by jmazzitelli** [comment]
> Can i raise the new clean PR ?

yes.
